### ISSUE TITLE
[silx.math.combo] min_max

### ISSUE THREAD

**Issue by payno** [silx-kit/silx#1261]
If I open a terminal and go for:

```
import numpy
from silx.math.combo import min_max

l = numpy.arange(10, numpy.int64)
min_max(l) 
```
This raises 
```

ERROR:silx.gui.qt._qt:<class 'TypeError'> No matching signature found   File "C:\Users\opdau\Documents\payno\silx\build\lib.win-amd64-3.5\silx\gui\plot\ColorBar.py", line 213, in _activeImageChanged
    data=image)
  File "C:\Users\opdau\Documents\payno\silx\build\lib.win-amd64-3.5\silx\gui\plot\ColorBar.py", line 159, in setColormap
    data=data)
  File "C:\Users\opdau\Documents\payno\silx\build\lib.win-amd64-3.5\silx\gui\plot\ColorBar.py", line 389, in setColormap
    self.colorScale.setColormap(colormap, data)
  File "C:\Users\opdau\Documents\payno\silx\build\lib.win-amd64-3.5\silx\gui\plot\ColorBar.py", line 518, in setColormap
    self.vmin, self.vmax = self._colormap.getColormapRange(data=data)
  File "C:\Users\opdau\Documents\payno\silx\build\lib.win-amd64-3.5\silx\gui\plot\Colormap.py", line 251, in getColormapRange
    min_, max_ = min_max(data, min_positive=False, finite=True)
  File "silx/math/combo.pyx", line 320, in silx.math.combo.min_max (silx\math\combo.c:14195)
    return _min_max(data, min_positive)
  File "silx/math/combo.pyx", line 117, in silx.math.combo.__pyx_fused_cpdef (silx\math\combo.c:3524)
    def _min_max(_number[::1] data, bint min_positive=False):

```

This append in our windows PC but work on other console. Seems like he is not handling numpy.int64 and in some examples replace it for int32.

**Comment by t20100** [comment]
This comes from the fused type `_number` which does not use sized types.

**Comment by t20100** [comment]
closed by #1262
